web_client/panels/AnnotationSelector.js

```diff
@@ -307,6 +311,11 @@ var AnnotationSelector = Panel.extend({
 
     _changeAnnotationHighlight(model) {
         this.$(`.h-annotation[data-id="${model.id}"]`).toggleClass('h-highlight-annotation', model.get('highlighted'));
+    },
+
+    _changeGlobalOpacity() {
+        this._opacity = this.$('#h-annotation-opacity').val();
```

Comment:
I'd like to see the opacity value somewhere.  A simple way would be to add a title attribute: `this.$('#h-annotation-opacity').attr('title', (100.0 * this._opacity).toFixed(0) + '%');`.  This would only show when hovering, not dragging, so there might be a better way to do this.